docs/docs/install/docker-compose.md

```diff
@@ -8,9 +8,13 @@ Docker Compose is the recommended method to run Immich in production. Below are
 
 ### Step 1 - Download the required files
 
-Download [`docker-compose.yml`][compose-file] [`example.env`][env-file].
+Create a directory of your choice (e.g. `./immich-app`) to hold the `docker-compose.yml` and `.env` files.
 
-From a directory of your choice (e.g. `./immich-app`) run the following commands:
+Download [`docker-compose.yml`][compose-file] and [`example.env`][env-file], either by running the following commands:
+
+```bash title="Move to the directory you created"
+cd ./immich-app     #(Use the actual directory you created)
+```
 
 ```bash title="Get docker-compose.yml file"
 wget https://github.com/immich-app/immich/releases/latest/download/docker-compose.yml
```

Comment:
This can be one code-block. I'm also not sure if it's even necessary to specify the `cd` command.

Reply:
>I'm also not sure if it's even necessary to specify the `cd` command.

Good point. I was just being dumb I guess, but I got stuck at that part because I missed the cd step